gradle/upstream-repositories.gradle

```diff
@@ -18,17 +22,17 @@ if (!hasProperty('upstreamBranch')) {
 		ext.upstreamBranch = 'release_' + version
 }
 
-def jenkinsPipelineRepo = { jobName -> "http://services.typefox.io/open-source/jenkins/job/$jobName/job/$upstreamBranch/lastStableBuild/artifact/build/maven-repository/" }
+def jenkinsPipelineRepo = { jobName, upstreamBranch -> "$JENKINS_URL/job/$jobName/job/$upstreamBranch/lastStableBuild/artifact/build/maven-repository/" }
 
 repositories {
 	jcenter()
 	// We have dependencies to test projects from xtext-extras, so Jenkins snapshots are the default
 	if (!hasProperty('useJenkinsSnapshots') || findProperty('useJenkinsSnapshots') == 'true') {
-		maven { url "http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository/" }
-		maven { url jenkinsPipelineRepo('xtext-lib') }
-		maven { url jenkinsPipelineRepo('xtext-core') }
-		maven { url jenkinsPipelineRepo('xtext-extras') }
-		maven { url jenkinsPipelineRepo('xtext-xtend') }
+		maven { url jenkinsPipelineRepo('lsp4j','master') }
```

Comment:
this does break since upstreamBranch will never be there on lsp4j

Reply:
The upstream branch for lsp4j will never be replaced. It will stay master.

Reply:
you dont understand: -PupstreamBranch=xxxxx

Reply:
Ah, right. I though you meant the adjustPipelines script. That would take care of lsp4j.
Makes sense now. I will revert for lsp4j. Good catch!